api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java

```diff
@@ -2199,21 +2183,25 @@ private void enableCovid19() {
     }
 
     private void enableOvc() {
+        configureBasicProgramDashboard(OVCProgram.OVC);
+
         extensions.add(visitAction(Extensions.OVC_INITIAL_VISIT_ACTION,
                 "ui.i18n.EncounterType.name." + EncounterTypes.OVC_INTAKE.uuid(),
                 "fas fa-fw fa-child",
                 "link",
                 enterStandardHtmlFormLink(PihCoreUtil.getFormResource("ovcIntake.xml")),
                 null,
-                visitDoesNotHaveEncounterOfType(EncounterTypes.OVC_INTAKE)));
+                and(or(patientAgeUnknown(), patientYoungerThan(22)),
+                        visitDoesNotHaveEncounterOfType(EncounterTypes.OVC_INTAKE))));
```

Comment:
For HIV Intake and Followup I think we don't allow an Intake if there is either a Intake or a Followup on the same visit... (but maybe that's not the same requirement here, I haven't checked, but wanted to throw it out there).

Reply:
That's sensible enough. The requirement is actually "no followup until there's an intake, no intake after there's already an intake," but we don't support those kinds of conditions (i.e., conditions across all visits/encounters).